### DISCUSSION TITLE
How do I set osu related files (.osk .osz and others) to launch automatically with stable instead of lazer?

### DISCUSSION THREAD

**Discussion by crumpledmint** [ppy/osu#29307] [Q&A]
Basically title

**Comment by kstefanowicz** [comment]
Assuming you're on Windows - you can go to **Settings > Apps > Default Apps > Choose defaults by file type**. This allows you to search for specific file types and set what application to launch in:
![image](https://github.com/user-attachments/assets/47a4c25c-b9e9-4bed-bd0e-7230b832eb73)

Clicking on a file type will open a pop-up "Select a default app for [filetype] files", that will let you choose what program to open in:
![image](https://github.com/user-attachments/assets/0c8ffd8b-dc3d-4cd1-9486-66b634e038c3)

One of those "osu!" apps is Stable, the other is Lazer. If switching between them doesn't seem to be working, you can manually direct it to `osu!.exe` by scrolling down on that pop-up and choosing "Choose an app on your PC":
![image](https://github.com/user-attachments/assets/faa68b5a-12f0-4e17-a1b8-a188cdfb3dd9)

Hope this helps!!
